hs/src/Reach/UnrollLoops.hs

```diff
@@ -179,8 +179,8 @@ instance Unroll LLStep where
       LLS_ToConsensus at send <$> ul recv <*> ul mtime
 
 instance Unroll LLProg where
-  ul (LLProg at opts ps dli s) =
-    LLProg at opts ps dli <$> ul s
+  ul (LLProg at opts ps dli dex s) =
```

Comment:
And unroll them, because we're going to verify them and the verifier can't deal with loops